docs/sources/installation/centos.md

```diff
@@ -6,153 +6,171 @@ page_keywords: Docker, Docker documentation, requirements, linux, centos, epel,
+				executable that produces the output you are currently reading.
+		 4. The Docker daemon streamed that output to the Docker client, which sent it
+				to your terminal.
 
-    $ sudo yum -y remove docker
+		To try something more ambitious, you can run an Ubuntu container with:
+		 $ docker run -it ubuntu bash
 
-Next, let's install the `docker-io` package which will install Docker on our host.
+		For more examples and ideas, visit:
+		 http://docs.docker.com/userguide/
 
-    $ sudo yum install docker-io
+  This command downloads a test image and runs it in a container.
+ 
+## Create a docker group		
 
-Please continue with the [Starting the Docker daemon](#starting-the-docker-daemon).
+The `docker` daemon binds to a Unix socket instead of a TCP port. By default
+that Unix socket is owned by the user `root` and other users can access it with
+`sudo`. For this reason, `docker` daemon always runs as the `root` user.
 
-### Uninstallation
+To avoid having to use `sudo` when you use the `docker` command, create a Unix
+group called `docker` and add users to it. When the `docker` daemon starts, it
+makes the ownership of the Unix socket read/writable by the `docker` group.
 
-To uninstall the Docker package:
+>**Warning**: The `docker` group is equivalent to the `root` user; For details
+>on how this impacts security in your system, see [*Docker Daemon Attack
+>Surface*](/articles/security/#docker-daemon-attack-surface) for details.
 
-    $ sudo yum -y remove docker-io
+To create the `docker` group and add your userwhi:
 
-The above command will not remove images, containers, volumes, or user created
-configuration files on your host. If you wish to delete all images, containers,
-and volumes run the following command:
+1. Log into Red Hat as a user with `sudo` privileges.
 
-    $ rm -rf /var/lib/docker
+    This procedure assumes you log in as the `ubuntu` user.
 
-You must delete the user created configuration files manually.
+3. Create the `docker` group and add your user.
 
-## Manual installation of latest Docker release
+    `sudo usermod -aG docker your_username`
 
-While using a package is the recommended way of installing Docker,
-the above package might not be the current release version. If you need the latest
-version, [you can install the binary directly](
-https://docs.docker.com/installation/binaries/).
+3. Log out and log back in.
 
-When installing the binary without a package, you may want
-to integrate Docker with Systemd. For this, install the two unit files
-(service and socket) from [the GitHub
-repository](https://github.com/docker/docker/tree/master/contrib/init/systemd)
-to `/etc/systemd/system`.
+    This ensures your user is running with the correct permissions.
 
-Please continue with the [Starting the Docker daemon](#starting-the-docker-daemon).
+4. Verify your work by running `docker` without `sudo`.
 
-## Starting the Docker daemon
+			$ docker run hello-world
```

Comment:
This line uses tabs, not spaces.

For consistency with the lines above, use back-ticks?